### DISCUSSION TITLE
Cluster Endpoint jdbc:mysql:aws vs jdbc:mysql

### DISCUSSION THREAD

**Discussion by narayananl23** [awslabs/aws-mysql-jdbc#359] [Q&A] [answered]
Is there any benefit or drawback to use jdbc:mysql:aws vs jdbc:mysql in the Cluster Endpoint

jdbc:mysql:aws://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:3306
Vs
jdbc:mysql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:3306

Both works.. no issues.

In another discussion, I see below. https://github.com/awslabs/aws-mysql-jdbc/discussions/181 

 "the AWS JDBC Driver for MySQL only supports failover functionality for the "jdbc:mysql:aws://" protocol."
 
 If I user "jdbc:mysql", does it not failover smoothly ?

**Comment by karenc-bq** [comment, marked as answer]
Hi @narayananl23, there will be no failover functionality when using the "jdbc:mysql" protocol.
The driver behaves in the exact same way as the [community MySQL driver](https://github.com/mysql/mysql-connector-j) when using the `jdbc:mysql` protocol.

**Comment by davecramer** [comment]
@narayananl23 The way that DriverManager works if you use `jdbc:mysql` the mysql driver will load. Not this driver. So the code from the aws-mysql-jdbc driver would not even run.
